src/lib/rtc-to-net/rtc-to-net.ts

```diff
@@ -330,6 +330,11 @@ import ProxyConfig = require('./proxyconfig');
     private channelSentBytes_ :number = 0;
     private channelReceivedBytes_ :number = 0;
 
+    //used to caculate the total bandwidth of this session in 5 second intervals
+    private stopBandwidthCalc_:number = 0;
+    private currBytesSession_:number = 0;
```

Comment:
Good to mention in the comment that this records bytes received and sent to/from the peer.